llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -194,6 +194,7 @@ cc_library(
     }),
     hdrs = glob([
         "include/llvm/Support/*.h",
+        "include/llvm/Support/FileSystem/*.h",
```

Comment:
Another option would be just globbing for "include/llvm/Support/**.h" which I think wouldn't be too unreasonable.

Reply:
I'm good with either.

Reply:
Decided to go with the globbier glob as likely to be more future-proof